### PR TITLE
Improved Risk description for Login CSRF

### PR DESCRIPTION
**Pull request by malikashish8**
This PR covers issue #131

### CODE REVIEW

**Comment by righettod** [comment]
Thank you very much for the PR, we will review it ASAP 😃

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ We recommend researching if the framework you are using has an option to achieve
...
 
-With a CSRF vulnerability on login form, an attacker can make a victim login as them and learn behavior from their searches. For more information about login CSRF and other risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
+With a CSRF vulnerability on login form, an attacker can make a victim login with an account they control. The attacker can than learn behavior from their interaction with the site with features such as search/activity/navigation history. The victim might also end up trusting the site to perform operations such as providing highly confidential information such as payment details assuming that they are logged in to their account. For more information about login CSRF and other risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
```
Comment (righettod) [inline cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: The PDF referenced is very old (2008 according to PDF properties and this [link](http://www.adambarth.com/papers/2008/barth-jackson-mitchell-b.pdf)): One of the example given is related to *iGoogle* and this service [is not active anymore](https://en.wikipedia.org/wiki/IGoogle).

A more recent and up-to-date source regarding *Login CSRF* must be added with a description of the attack flow in order to allow the dev team to better understand the security issue.

The content added is a kind of resume of the content of the section 3 of the PDF but the explanation of the PDF allow to better understand the different scenario in which *Login CSRF* can be applied, so for me, the actual section invite the reader to read the section 3 of the PDF and this one contains well explanation.

@ThunderSon @mackowski What do you think?

Comment (manideepkonakandla) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Sounds good to me. "For more information on login CSRF and it's associated risks" might be a better wording

**Comment by righettod** [comment]
Waiting review from @ThunderSon and @mackowski before to go further.

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md (hunk 1 above)
Comment (mackowski) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: I agree with @righettod. Basically PDF describes login CSRF very well. 
I think that if there will be a problems with finding newer reference we can put here better and wider description of Login CSRF, based on that PDF. And reference that this description is based on this paper. 

So I think that we should either not try to explain it here but only invite the reader to read the section 3 of the PDF (or newer reference) OR try to rewrite it in a way that reader will be able to understand it well.

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ We recommend researching if the framework you are using has an option to achieve
...
 
-With a CSRF vulnerability on login form, an attacker can make a victim login as them and learn behavior from their searches. For more information about login CSRF and other risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
+With a CSRF vulnerability on login form, an attacker can make a victim login with an account they control. The attacker can then learn behaviour from their interaction with the site with features such as search/activity/navigation history. The victim might also end up trusting the site to perform operations such as providing highly confidential information such as payment details assuming that they are logged into their account. For more information about login CSRF and it's associated risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
```
Comment (ThunderSon) [inline cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: As I have spotted several English mistakes, I thought of refactoring it for user readability. Let me know what you think:

Having a CSRF vulnerability on the login form, an attacker can lure a victim to login with an account that the attacker controls. The user’s behavior would then be studied by the actions taken, and if that application contains highly confidential information (_e.g._ payment details), those too will be compromised. For additional information on this attack, refer to this [paper](https://seclab.stanford.edu/websec/csrf/csrf.pdf).

**Comment by righettod** [comment]
> ThunderSon: Should we add that paper to our repository? That way we won't be in danger of losing it if at some point the website was taken down.

Agree, I propose to named it `Cross-Site_Request_Forgery_Prevention_Cheat_Sheet_Login_CSRF.pdf`.

> mackowski: So I think that we should either not try to explain it here but only invite the reader to read the section 3 of the PDF (or newer reference) OR try to rewrite it in a way that reader will be able to understand it well.

Agree, I propose that we should invite the reader to read the section 3, it's short and well expained, and we will just mention the `Login CSRF` itself as a kind of CSRF attack.

**Comment by righettod** [comment]
Hello,

In order to make the project going forward and no keep PR open for a too long time,  we will close this PR next week if no update are pushed.

Thank you very much in advance for your understanding 😃

**Comment by righettod** [comment]
No feedback from the author from more than 1 month.

**Pull request closed without merging**